elements/auras.lua

```diff
@@ -194,25 +205,49 @@ local function updateIcon(element, unit, index, offset, filter, isDebuff, visibl
 
 		* show - indicates whether the aura button should be shown (boolean)
 		--]]
-		local show = (element.CustomFilter or customFilter) (element, unit, button, name, texture,
-			count, debuffType, duration, expiration, caster, isStealable, nameplateShowSelf, spellID,
-			canApply, isBossDebuff, casterIsPlayer, nameplateShowAll,timeMod, effect1, effect2, effect3)
-
-		if(show) then
+		local show = (element.CustomFilter or customFilter)(
+			element,
+			unit,
+			button,
+			name,
+			texture,
+			count,
+			debuffType,
+			duration,
+			expiration,
+			caster,
+			isStealable,
+			nameplateShowSelf,
+			spellID,
+			canApply,
+			isBossDebuff,
+			casterIsPlayer,
+			nameplateShowAll,
+			timeMod,
+			effect1,
+			effect2,
+			effect3
+		)
+
+		if show then
 			-- We might want to consider delaying the creation of an actual cooldown
 			-- object to this point, but I think that will just make things needlessly
 			-- complicated.
-			if(button.cd and not element.disableCooldown) then
-				if(duration and duration > 0) then
+			if button.cd and not element.disableCooldown then
+				if duration and duration > 0 then
 					button.cd:SetCooldown(expiration - duration, duration)
 					button.cd:Show()
 				else
 					button.cd:Hide()
 				end
 			end
 
-			if(button.overlay) then
-				if((isDebuff and element.showDebuffType) or (not isDebuff and element.showBuffType) or element.showType) then
+			if button.overlay then
+				if
+					(isDebuff and element.showDebuffType)
+					or (not isDebuff and element.showBuffType)
+					or element.showType
+				then
```

Comment:
This one decreases readability.

Reply:
This one is because the condition exceeded the max line length of 120 cols.

I do find those quite readable btw as it sorts the sub-conditions by precedence which is quite nice. It makes `A or B or C` quire obvious in this case.